### ISSUE TITLE
Compile error svn 11329

### ISSUE THREAD

**Issue by danyconnect** [oscam-emu/oscam-emu#223]
```
| Tools:
|  CROSS    = /opt/oscam/toolchains/rasb_hard/bin/arm-bcm2708hardfp-linux-gnueabi-
|  CC       = /opt/oscam/toolchains/rasb_hard/bin/arm-bcm2708hardfp-linux-gnueabi-gcc
| Settings:
|  CONF_DIR = /usr/local/etc
|  CC_OPTS  = -Os -ggdb -pipe -ffunction-sections -fdata-sections
|  CC_WARN  = -w
|  CFLAGS   = -fexpensive-optimizations -DWITH_LIBUSB=1
|  LDFLAGS  = -Wl,--gc-sections -DWITH_LIBUSB=1 -Wl,-rpath,/lib -Wl,-rpath,/usr/local/lib -Wl,-rpath,/usr/lib -L/opt/oscam/toolchains/rasb_hard/arm-bcm2708hardfp-linux-gnueabi/sysroot/usr/lib
|  LIBS     = -lusb-1.0 -lrt -lpthread -ldl -lrt
|  UseFlags = USE_LIBUSB=1
| Config:
|  Addons   : WEBIF WEBIF_LIVELOG WEBIF_JQUERY TOUCH HAVE_DVBAPI READ_SDT_CHARSETS IRDETO_GUESSING WITH_DEBUG MODULE_MONITOR WITH_LB CS_CACHEEX CW_CYCLE_CHECK CLOCKFIX IPV6SUPPORT WITH_EMU
|  Protocols: CAMD35 CAMD35_TCP NEWCAMD CCCAM CCCSHARE CONSTCW
|  Readers  : NAGRA IRDETO CONAX CRYPTOWORKS SECA VIACCESS VIDEOGUARD DRE TONGFANG BULCRYPT GRIFFIN DGCRYPT
|  CardRdrs : PHOENIX INTERNAL SMARGO SMART
|  Compiler : arm-bcm2708hardfp-linux-gnueabi-gcc (crosstool-NG 1.15.2) 4.7.1 20120402 (prerelease)
|  Config   : build/arm-bcm2708hardfp-linux-gnueabi-libusb/config.mak
|  Binary   : ../archiv/oscam-svn11329-rasb_hard-webif-libusb-oscam-emu-patched
+-------------------------------------------------------------------------------
HOSTCC	webif/pages_gen
GEN	webif/pages.c
GEN	Compressed 416560 template bytes into 172698 bytes. 243862 saved bytes (58.54%).
CC	cscrypt/bn_exp.c
CC	cscrypt/bn_add.c
CC	cscrypt/bn_mul.c
CC	cscrypt/aes.c
CC	cscrypt/bn_ctx.c
CC	cscrypt/bn_asm.c
CC	cscrypt/bn_lib.c
CC	cscrypt/bn_div.c
CC	cscrypt/bn_print.c
CC	cscrypt/bn_shift.c
CC	cscrypt/bn_sqr.c
CC	cscrypt/bn_word.c
CC	cscrypt/mem.c
CC	cscrypt/i_ecb.c
CC	cscrypt/i_cbc.c
CC	cscrypt/des.c
CC	cscrypt/i_skey.c
CC	cscrypt/md5.c
CC	cscrypt/rc6.c
CC	cscrypt/sha1.c
CC	csctapi/atr.c
CC	csctapi/icc_async.c
CC	csctapi/io_serial.c
CC	csctapi/protocol_t0.c
CC	csctapi/protocol_t1.c
CC	csctapi/ifd_phoenix.c
CC	csctapi/ifd_sci.c
CC	csctapi/ifd_smargo.c
CC	minilzo/minilzo.c
CC	csctapi/ifd_smartreader.c
CC	module-cacheex.c
CC	module-camd35-cacheex.c
CC	module-camd35.c
CC	module-cccam-cacheex.c
CC	module-cccam.c
CC	module-cccshare.c
CC	module-constcw.c
CC	module-emulator.c
CC	module-emulator-osemu.c
CC	module-emulator-stream.c
CC	ffdecsa/ffdecsa.c
CC	module-csp.c
CC	module-cw-cycle-check.c
CC	module-dvbapi-chancache.c
CC	module-dvbapi.c
CC	module-ird-guess.c
CC	module-monitor.c
CC	module-newcamd-des.c
CC	module-newcamd.c
CC	module-stat.c
CC	module-webif-lib.c
CC	module-webif-tpl.c
CC	module-webif.c
module-webif.c: In function 'send_oscam_config_streamrelay':
module-webif.c:1219:25: error: 'MNU_CFG_STREAMRELAY' undeclared (first use in this function)
module-webif.c:1219:25: note: each undeclared identifier is reported only once for each function it appears in
Makefile:437: recipe for target 'build/arm-bcm2708hardfp-linux-gnueabi-libusb/module-webif.o' failed
make[1]: *** [build/arm-bcm2708hardfp-linux-gnueabi-libusb/module-webif.o] Error 1
make[1]: *** Waiting for unfinished jobs....
Makefile:389: recipe for target 'all' failed
make: *** [all] Error 2``
```

**Comment by oscam-emu** [comment]
Will be fixed soon. No need for a new issue report every time.
